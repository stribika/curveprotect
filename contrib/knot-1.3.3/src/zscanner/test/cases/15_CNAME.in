$ORIGIN	.
$TTL	1

; The CNAME is the same as the NS, so there are the differences and basics only.

; OK
@	CNAME	test.example.com		; Relative dname
@	CNAME	\# 1 00				; Hexadecimal rdata
@	TYPE5	\# 1 00				; TYPE + Hexadecimal rdata
@	TYPE5	@				; TYPE
@	cname	@				; Type in lower-case

; KO
@	CNAME
