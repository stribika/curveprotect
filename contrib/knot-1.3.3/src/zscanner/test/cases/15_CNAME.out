OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0005
RDATA=0474657374076578616D706C6503636F6D00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0005
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0005
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0005
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0005
RDATA=00
------
WARNG=ZSCANNER_EBAD_RDATA
------
